pkg/api/controller.go

```diff
@@ -816,17 +816,20 @@ func (c *Controller) generateResetPasswordToken(email string) (string, error) {
 	return GenerateJWTResetPassword(secret, email, currentTime, currentTime.Add(DefaultResetPasswordExpiration))
 }
 
-func (c *Controller) inviteUserRequest(email string) error {
-	// TODO (@shimi9276) create template for sending the email with link for invite using the passed template
-	token, err := c.generateResetPasswordToken(email)
+func (c *Controller) inviteUserRequest(emailAddr string) error {
+	token, err := c.generateResetPasswordToken(emailAddr)
 	if err != nil {
 		return err
 	}
-	err = c.Emailer.SendEmailWithLimit([]string{email}, token, token, nil)
+	emailBody, err := email.ConstructInviteUserEmailTemplate(c.Emailer.Params.LakefsBaseURLEndpoint, token)
```

Comment:
The `email` package holds the template - so having the logic of sending an invite email should be there.
You can have one function that sends email based on template - so keep it general by passing args the same way the template engine passes arguments.
Capture the parameters that can be used by the email and call a function that send an invite - this function should format and send the email.